src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,51 @@
+var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
+var currencies = ['usd']
+
+function optionsChecker (options) {
+  if ('currency' in options) {
+    options.currency = options.currency || 'usd'
+    if (currencies.indexOf(options.currency) === -1) {
+      throw new Error('This is not a valid currency')
+    }
+  }
+  if ('timeBound' in options) {
+    options.timeBound = options.timeBound || ''
+    if (timeBound.indexOf(options.timeBound) === -1) {
+      throw new Error('This is not a valid timebound suffix')
+    }
+  }
+}
+
+function metricNameGenerator (metric, options) {
```

Comment:
Add documentation here that the `metric` name will get `_usd` and/or `_<number><[d|y]>` appended here, depending on the options